packages/@react-aria/focus/stories/FocusScope.stories.tsx

```diff
@@ -18,7 +18,7 @@ import ReactDOM from 'react-dom';
 const dialogsRoot = 'dialogsRoot';
 
 interface StoryProps {
-  usePortal: boolean,
+  isPortaled: boolean,
```

Comment:
moved away from usePortal because that sounds like a hook and this is just a boolean prop